### DISCUSSION TITLE
Filter and Sorting Feature in GitBook Tables

### DISCUSSION THREAD

**Discussion by lt-bha** [GitbookIO/community#384] [Feature Requests]
The improved tables are great! But please add a filter and sort function or option (filters would be very useful in the checkbox or select column type)

Thank you!

**Comment by Tommy-Ruse** [comment]
Yes! Even basic ascending/descending table sorting would be extremely helpful. Currently seems like embedding a google doc is the best (not great) option

**Reply by jumptheknot** [reply]
This is essential for large tables.

**Comment by Graciax8** [comment]
+100
